src/App.tsx

```diff
@@ -16,47 +22,84 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [goods, setGoods] = useState([...goodsFromServer]);
+  const [activeButton, setActiveButton] = useState<SortType | null>(null);
+  const [resetVisible, setResetVisible] = useState(false);
+
+  const handleSortAlphabetically = () => {
+    const sortedGoods = [...goods].sort();
+
+    setGoods(sortedGoods);
+    setActiveButton(SortType.Alphabetical);
+    setResetVisible(true);
+  };
+
+  const handleSortByLength = () => {
+    const sortedGoods = [...goods].sort((a, b) => a.length - b.length);
+
+    setGoods(sortedGoods);
+    setActiveButton(SortType.Length);
+    setResetVisible(true);
+  };
```

Comment:
so both of these handlers will be redundant.
You need only one handler that accept sortType and set it to state